frappe/patches/v14_0/update_multistep_webforms.py

```diff
@@ -4,6 +4,9 @@
 def execute():
 	frappe.reload_doctype("Web Form")
 
+	if not frappe.db.has_column("Web Form", "is_multi_step_form"):
+		return
```

Comment:
This can only happen for sites where this field never existed... I am guessing v13 -> v14? 🥲